docs/getting_started.md

```diff
@@ -1,187 +1,285 @@
+```
+
+### Install [OpenWhisk](https://github.com/apache/openwhisk)
+[OpenWhisk](https://github.com/apache/openwhisk) must be running to build Meteoroid.
 
 ```bash
 cd fiware-meteoroid/docker/openwhisk-devtools/docker-compose
 make quick-start
+```
 
+### Install Meteoroid core
+To install the Meteoroid using Docker (Docker Compose)
+
+```bash
+cd fiware-meteoroid/docker/
+docker-compose up -d
 ```
 
 ### Install [Meteoroid CLI](https://github.com/OkinawaOpenLaboratory/fiware-meteoroid-cli)
+To install the [Meteoroid CLI](https://github.com/OkinawaOpenLaboratory/fiware-meteoroid-cli) you can use pip.
+
+[Meteoroid CLI Documentation.](./cli.md)
 
 ```bash
 pip install meteoroid-cli
 ```
 
-### Install Meteoroid
+### (Option) Export METEOROID_SCHEMA_ENDPOINT for CLI
+**METEOROID_SCHEMA_ENDPOINT** is a specified URL to connect to Meteoroid core from Meteoroid CLI.
+Please change the host and port as needed.
 
-You can install Meteoroid in two ways:
+Default endpoint is **http://localhost:3000/schema/?format=corejson**
 
-- [Automatic installation](https://github.com/OkinawaOpenLaboratory/fiware-meteoroid/tree/master#automatic-installation) using Docker (Docker Compose)
-- [Manual installation](https://github.com/OkinawaOpenLaboratory/fiware-meteoroid/tree/master#manual-installation) using pipenv and Django manage.py
+```
+export METEOROID_SCHEMA_ENDPOINT=http://(host):(port)/schema/?format=corejson
+```
 
+
+## Usage
+
+### Create a Python script
+Create a Python script named **thermometer.py**
+The first argument of the main function is provided POST parameter and function param option.
+
+#### Parameters
+
+* **temperature**: The temperature is given an attribute of Entity as the HTTP request parameter(POST) assumed structures of Orion notification.
+* **threshold**: The threshold is given parameters when creating or updating function as a command line option.
+
+[Support Language](./user_and_programmers_manual/support_languages.md)
+
+```python
+def main(params):
+    temperature = params['data'][0]['temperature']['value']
+    threshold = int(params['threshold'])
+    message = "No problem."
+    if temperature > threshold:
+        message = "Dangerous because it’s too hot!"
+    print(temperature, message)
+    return {"temperature": temperature, "message": message}
+```
 ---
 
-## Automatic installation
-using Docker (Docker Compose)
+### Create Function
+Create a function with the name **Thermometer**.
+You can pass parameters to Function with the **param** option.
+
+`meteoroid function create [name] [file path] --language [language:version] --param [key] [value]`
 
 ```bash
-cd fiware-meteoroid/docker/
-docker-compose up -d
+meteoroid function create Thermometer thermometer.py --language python:3 --param threshold 35
 ```
 
-### Export METEOROID_SCHEMA_ENDPOINT for CLI (Option)
-Defualt endpoint (http://localhost:3000/schema/?format=corejson)
+result:
```

Comment:
`Output:` is good because `result:` is confused with Result API.

Reply:
OK. I have fixed it.